### DISCUSSION TITLE
In colorblind mode draft PR and a closed PR are indistinguishable

### DISCUSSION THREAD

**Discussion by elichai** [community/community#15194] [Accessibility]
![Screenshot_20220419_175502](https://user-images.githubusercontent.com/2167860/164033164-3df57120-348f-4101-a792-4bf76873e88e.png)
The first PR is a draft, the last PR is closed, the color of both looks exactly the same.
(it is even more confusing because I'm already used to before the feature where open was blue, closed was red and the draft was grey)

(I have Deuteranopia)

**Comment by suprstarrd** [comment]
Closed here is orange and drafts are gray. I'm more surprised open PR's aren't orange!

**Comment by elichai** [comment]
I'm confused hehe,
Here's a better example, with the feature on:
![Screenshot_20220419_180831](https://user-images.githubusercontent.com/2167860/164036042-ec89d842-16cc-4742-b539-87f6c1e703f3.png)

Here `A` is blueish and means *Merged*, `B` is reddish and means *Open*, `C` is grey and means *Draft*, and `D` is also grey and means `Closed`.

With the feature off:
![Screenshot_20220419_181227](https://user-images.githubusercontent.com/2167860/164036758-a274696a-9d15-476b-8e3d-d9f6d7c358ca.png)
Here `A` is blue/purplish and means merged, `B` is blue similar to `A` but not purplish (confusing, but at least they have a very different icon), `C` is the same grey, and `D` is red.

**Comment by cehfisher** [comment]
Hello @elichai - thank you for providing this feedback! We wanted to follow up on this issue. Have the recent updates to the [high contrast and colorblind themes](https://github.com/settings/appearance) resolved this issue for you or are you still seeing the same issues as before? Thank you!

**Comment by cehfisher** [comment]
Adding images of current colorblindness-focused theme. It looks like the original issue has been resolved.

Light Protanopia & Deuteranopia
![Light Protanopia   Deuteranopia](https://github.com/user-attachments/assets/bcbfde06-6937-4955-b615-4ab8caacd5da)

Light Tritanopia
![Light Tritanopia](https://github.com/user-attachments/assets/a426c46b-440b-49e3-8b5d-673dba064497)

Dark Protanopia & Deuteranopia
![Dark Protanopia   Deuteranopia](https://github.com/user-attachments/assets/6bac0254-5f97-4fb2-992e-92e5b39be1dd)

Dark Tritanopia
![Dark Tritanopia](https://github.com/user-attachments/assets/1a5333da-8572-4a91-a6ad-155a488a5099)
